### PR TITLE
New template to create HDInsight cluster with Hue app on Edge Node

### PR DESCRIPTION
**Pull request by ezubatov**
Created a new template to setup HDIcluster and install the Hue application on edge node.
Modified azuredeploy.json file for generic template (which sets up cluster and edge node) so it calls app installation script with new set of parameters.
Generic template is located here: hdinsight-linux-with-edge-node
New template is located here: hdinsight-linux-with-hue-on-edge-node

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@ezubatov__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

File: hdinsight-linux-with-hue-on-edge-node/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+        {
+            "name": "HDInsightDeployment",
+            "type": "Microsoft.Resources/deployments",
+            "apiVersion": "[variables('defaultApiVersion')]",
+            "dependsOn": [
+            ],
+            "properties": {
+                "mode": "Incremental",
+                "templateLink": {
+                    "uri": "https://raw.githubusercontent.com/ezubatov/azure-quickstart-templates/master/hdinsight-linux-with-edge-node/azuredeploy.json",
```
Comment (travismc1) [inline hdinsight-linux-with-hue-on-edge-node/azuredeploy.json]: This should point to the azure repo

File: hdinsight-linux-with-hue-on-edge-node/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+                    "clusterLoginUserName": { "value": "[parameters('clusterLoginUserName')]" },
+                    "clusterLoginPassword": { "value": "[parameters('clusterLoginPassword')]" },
+                    "sshUserName": { "value": "[parameters('sshUserName')]" },
+                    "sshPassword": { "value": "[parameters('sshPassword')]" },
+                    "clusterStorageAccountName": { "value": "[parameters('clusterStorageAccountName')]" },
+                    "clusterWorkerNodeCount": { "value": "[parameters('clusterWorkerNodeCount')]" },
+                    "edgeNodeVhdStorageAccountName": { "value": "[parameters('edgeNodeVhdStorageAccountName')]" },
+                    "edgeNodeVirtualMachineName": { "value": "[parameters('edgeNodeVirtualMachineName')]" },
+                    "edgeNodeVirtualMachineSize": { "value": "[parameters('edgeNodeVirtualMachineSize')]" },
+                    "edgeNodeSetupScriptUri": { "value": "https://raw.githubusercontent.com/ezubatov/azure-quickstart-templates/master/hdinsight-linux-with-hue-on-edge-node/install-hue-edge-v01.sh" }
```
Comment (travismc1) [inline hdinsight-linux-with-hue-on-edge-node/azuredeploy.json]: This should also point to the azure repository. Just replace ezubatov with Azure

**Comment by singhkay** [comment]
lgtm

**Pull request merged**